### PR TITLE
[docs] CLI commands ifconfig - linkmetrics

### PR DESCRIPTION
**Pull request by sarah-iot**

### CODE REVIEW

File: src/cli/cli.cpp [hunk 1]
```diff
@@ -4001,11 +4331,54 @@ otError Interpreter::ProcessLinkMetricsMgmt(Arg aArgs[])
...
+         * @code
+         * > linkmetrics mgmt fe80:0:0:0:3092:f334:1455:1ad2 enhanced-ack register qm r
+         * Done
+         * > Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
+         * Status: Cannot support new series
+         * @endcode
+         * @cparam linkmetrics mgmt @ca{peer-ipaddr} enhanced-ack register [@ca{qmr}][@ca{r}]
+         * [`q`, `m`, and `r`] map to #otLinkMetricsValues. At most, two options are allowed to
+         * select (per spec 4.11.3.4.4.6).
+         * - `q`: Layer 2 LQI.
```
Comment (Vyrastas) [inline src/cli/cli.cpp:4370]: "two options are allowed to select" sounds awkward, can you rephrase?

File: src/cli/cli.cpp [hunk 2]
```diff
@@ -3945,6 +4249,32 @@ otError Interpreter::ProcessLinkMetricsMgmt(Arg aArgs[])
...
+     * linkmetrics mgmt fe80:0:0:0:3092:f334:1455:1ad2 forward 1 dra pqmr
+     * Done
+     * > Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
+     * Status: SUCCESS
+     * @endcode
+     * @cparam linkmetrics mgmt @ca{peer-ipaddr} forward @ca{series-id} [@ca{ldraX}][@ca{pqmr}]
+     * - `peer-ipaddr`: Peer address.
+     * - `series-id`: The Series ID.
+     * - [`l`, `d`, `r`, and `a`] map to #otLinkMetricsSeriesFlags. `X` represents none of the
+     *   `otLinkMetricsSeriesFlags`, and indicates to stop accounting and remove the series.
```
Comment (Vyrastas) [inline src/cli/cli.cpp:4264]: ...and stops the accounting and removes the series.

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Just two minor things

File: src/cli/cli.cpp (hunk 1 above)
Comment (sarah-iot) [reply src/cli/cli.cpp]: Sure, done.

File: src/cli/cli.cpp (hunk 2 above)
Comment (sarah-iot) [reply src/cli/cli.cpp]: Thanks, done.

**Review by abtink — APPROVED** [review]
Looks great. Thanks.

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]
